### PR TITLE
[4.0] workflow namespace

### PR DESCRIPTION
**Pull request by brianteeman**
Quick pr to namespaace jtext and jtml

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/states/default.php [hunk 1]
```diff
@@ -25,7 +28,7 @@
 if ($saveOrder)
 {
 	$saveOrderingUrl = 'index.php?option=com_workflow&task=states.saveOrderAjax&' . JSession::getFormToken() . '=1';
-	JHtml::_('draggablelist.draggable');
+	HTMLHelper_('draggablelist.draggable');
```
Comment (C-Lodder) [inline administrator/components/com_workflow/tmpl/states/default.php]: `HTMLHelper_` >> `HTMLHelper::_`.

Same for the rest of them

Comment (brianteeman) [reply administrator/components/com_workflow/tmpl/states/default.php]: do thats what happens when you do it quickly

Comment (brianteeman) [reply administrator/components/com_workflow/tmpl/states/default.php]: fixed it now

**Comment by laoneo** [comment]
:+1:

**Comment by brianteeman** [comment]
Thanks for merging but it was a waste of all our time #21476

**Pull request merged**